tests/render/fixtures/BasicDatasetProfiler_evrs.json

```diff
@@ -308,7 +308,7 @@
         "type": "expect_column_values_to_be_in_set",
         "kwargs": {
           "column": "Unnamed: 0",
-          "value_set": [],
+          "value_set": ["a", "b", "c"],
```

Comment:
Just blanketly applying this value set to any null or empty cases - there are a lot of instances in our renderer smoketests but it should be relatively straightforward.